OpenRA.Mods.Common/Traits/Render/RenderDetectionCircle.cs

```diff
@@ -19,14 +19,29 @@ namespace OpenRA.Mods.Common.Traits
 {
 	class RenderDetectionCircleInfo : ITraitInfo, Requires<DetectCloakedInfo>
 	{
-		public object Create(ActorInitializer init) { return new RenderDetectionCircle(init.Self); }
+		[Desc("Draw radar update line")]
```

Comment:
`Draw line from this actor to the detection circle` or similar please, these descriptions don't offer anything over the field names as-is.